graphics/src/Material.cc

```diff
@@ -119,6 +123,7 @@ Material::Material(const math::Color &_clr)
     std::to_string(this->dataPtr->counter++);
   this->dataPtr->blendMode = REPLACE;
   this->dataPtr->shadeMode = GOURAUD;
+  this->dataPtr->renderOrder = 0;
```

Comment:
This is already set during the declaration, do you need to set it again in the constructors?